.storybook/preview-head-vrt.html

```diff
@@ -6,4 +6,11 @@
 		animation-duration: 0ms !important;
 		transition-duration: 0ms !important;
 	}
+
+	#root .googlesitekit-vrt-animation-none *,
+	#root .googlesitekit-vrt-animation-none *::before,
+	#root .googlesitekit-vrt-animation-none *::after {
+		animation: none !important;
+		transition: none !important;
+	}
```

Comment:
These are somewhat obscure selectors with important modifiers. I'm not opposed to `!important`, but my usual rule around using it is to document _why_, so the next person who encounters it doesn't think "oh, is this even needed?" and then remove it or wonder what it's for 😅 

Can you add a comment here explaining what this does?